### ISSUE TITLE
Switch from api.rekor.dev to api.sigstore.dev.

### ISSUE THREAD

**Issue by lukehinds** [sigstore/sigstore#53]

**Comment by bobcallaway** [comment]
do we want api.sigstore.dev or rekor.sigstore.dev?

**Comment by lukehinds** [comment]
I agree rekor.sigstore.dev is more descriptive, this issue is just to follow suit on:

https://github.com/sigstore/cosign/commit/b48428df5d56433c471f0ec1ea1d3567e7143283

However I can't see this change in the public good instance, where are with this at present @dlorenc ?
